templates/fine-tune-llm/training_configs/llama-2-70b-2k-4xg5_48xlarge.yaml

```diff
@@ -0,0 +1,17 @@
+model_id: meta-llama/Llama-2-70b-hf # <-- change this to the model you want to fine-tune
+train_path: s3://air-example-data/gsm8k/train.jsonl # <-- change this to the path to your training data
+valid_path: s3://air-example-data/gsm8k/test.jsonl # <-- change this to the path to your validation data. This is optional
+context_length: 2048 # <-- change this to the context length you want to use
+num_devices: 32
+num_epochs: 10 # <-- change this to the number of epochs that you want to train for
+train_batch_size_per_device: 16
+eval_batch_size_per_device: 16
+learning_rate: 5e-6
+num_checkpoints_to_keep: 1
+output_dir: /mnt/local_storage
+dataset_size_scaling_factor: 1000
```

Comment:
what's this?

Reply:
For public finetuning, we enforce a limit on the size of dataset. This is basically a placeholder value to make sure the limit is always large enough for private finetuning

Reply:
maybe we should add a comment to explain